lib/galaxy/datatypes/test/gsas_instrument.prm

```diff
@@ -0,0 +1,12 @@
+            123456789012345678901234567890123456789012345678901234567890
+INS   BANK      1                                                               
+INS   HTYPE   PNCR                                                              
+INS  1 ICONS  1.909000  0.000000      -0.1         0       0.0    0       0.0
+INS  1I HEAD  DUMMY INCIDENT SPECTRUM FOR DIFFRACTOMETER D1A                    
+INS  1I ITYP    0    0.0000  180.0000         1                                 
+INS  1PRCF1     1    6      0.01                                                
+INS  1PRCF11   0.354031E+03  -0.760404E+03   0.651592E+03   0.000000E+00        
+INS  1PRCF12   0.000000E+00   0.000000E+00   0.000000E+00   0.000000E+00        
+INS  1PRCF2     2    6      0.01                                                
+INS  1PRCF21   0.354031E+03  -0.760404E+03   0.651592E+03   0.000000E+00        
+INS  1PRCF22   0.000000E+00   0.000000E+00                                      
```

Comment:
```suggestion
INS  1PRCF22   0.000000E+00   0.000000E+00                                      

```

Missing trailing newline leading to test failure: https://github.com/MaterialsGalaxy/neutron_galaxy/actions/runs/11329142116/job/31503989328?pr=1